.travis.yml

```diff
@@ -58,6 +58,10 @@ matrix:
       env: SONATA_BLOCK='dev-master as 3'
     - php: '7.1'
       env: SYMFONY_DEPRECATIONS_HELPER=0
+    - php: '7.1'
+      env: SYMFONY_VERSION=^3.0
+    - php: '7.1'
+      env: DEPENDENCIES=beta
```

Comment:
We manage this file with https://github.com/Soullivaneuh/sonata-dev-kit

Reply:
https://github.com/sonata-project/dev-kit